crates/assemble/src/ops/mod.rs

```diff
@@ -23,8 +22,7 @@ macro_rules! gen_schema {
 
 /// Adds ops collections to the given partial built catalog. The tables will be modified in place
 /// to add the resources required for the ops (logs and stats) collections.
-pub fn generate_ops_collections(tables: &mut tables::Sources) {
-    let prefixes = all_top_level_prefixes(&*tables);
+pub fn generate_ops_collections(tables: &mut tables::Sources, dataplane: &str) {
```

Comment:
just a heads up, this `ops` module is essentially deprecated and we'll want to rip it all out once we get planned reporting rolled out, so any changes don't need to be pretty.